debian/kafka-streams-examples/include/etc/confluent/docker/ensure

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset \
+    -o errexit \
+    -o verbose \
+    -o xtrace
+
+echo "===> Verify that Kafka is healthy ..."
+
+if [[ -n "${STREAMS_SECURITY_PROTOCOL-}" ]] && [[ $STREAMS_SECURITY_PROTOCOL="SSL" ]]
+then
+
+    cub kafka-ready \
+        "${STREAMS_CUB_KAFKA_MIN_BROKERS:-1}" \
+        "${STREAMS_CUB_KAFKA_TIMEOUT:-40}" \
+        -b "$STREAMS_BOOTSTRAP_SERVERS" \
+        --config /etc/"${COMPONENT}"/kafka-streams-examples.properties
```

Comment:
Ah, I see.  In that case I'd be happy for any guidance on what I should do here.  This part of the code I am not feeling very confident about (notably because I couldn't figure out how to test this).